tools/components.js

```diff
@@ -0,0 +1,46 @@
+export default [
+  'Accordion',
+  'Affix',
+  'Alert',
+  'Badge',
+  'Button',
+  'ButtonGroup',
+  'ButtonToolbar',
+  'Carousel',
+  'CarouselItem',
+  'Col',
+  'CollapsableNav',
+  'DropdownButton',
+  'DropdownMenu',
+  'FormGroup',
+  'Glyphicon',
+  'Grid',
+  'Input',
+  'Interpolate',
+  'Jumbotron',
+  'Label',
+  'ListGroup',
+  'ListGroupItem',
+  'MenuItem',
+  'Modal',
+  'ModalTrigger',
+  'Nav',
+  'Navbar',
+  'NavItem',
+  'OverlayTrigger',
+  'PageHeader',
+  'PageItem',
+  'Pager',
+  'Panel',
+  'PanelGroup',
+  'Popover',
+  'ProgressBar',
+  'Row',
+  'SplitButton',
+  'SubNav',
+  'TabbedArea',
+  'Table',
+  'TabPane',
+  'Tooltip',
+  'Well'
+];
```

Comment:
Was that function which checks if something was a component not sufficient? Is that why you needed to explicitly list them?